### PR TITLE
Transcript prototype

### PR DESCRIPTION
**Pull request by cmpich**

### CODE REVIEW

File: source/org/zfin/framework/dao/VocabularyDAO.java [hunk 1]
```diff
@@ -0,0 +1 @@
...

import org.zfin.framework.Vocabulary;

public class VocabularyDAO extends BaseSQLDAO<Vocabulary> {

    public VocabularyDAO() {
        super(Vocabulary.class);
    }

}
```
Comment (rtaylorzfin) [inline source/org/zfin/framework/dao/VocabularyDAO.java:1]: i'm seeing weird line-endings here.

File: source/org/zfin/db/postGmakePostloaddb/1154/ZFIN-transcript-ensembl.sql [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+(
+    vt_id           serial not null,
+    vt_name         text   not null,
+    vt_v_id         integer,
+    vt_date_created DATE   NOT NULL DEFAULT CURRENT_DATE
+);
+
+insert into vocabulary (v_name, v_description) VALUES ('transcript annotation method','All allowed values for transcript annotation method');
+
+insert into vocabulary_term (vt_name, vt_v_id) VALUES ('Havana', (select v_id from vocabulary));
```
Comment (rtaylorzfin) [inline source/org/zfin/db/postGmakePostloaddb/1154/ZFIN-transcript-ensembl.sql:61]: Does this work for getting the right ID from vocabulary?

File: source/org/zfin/framework/dao/VocabularyDAO.java (hunk 1 above)
Comment (rtaylorzfin) [reply source/org/zfin/framework/dao/VocabularyDAO.java:1]: I just created a PR to this PR for fixing the line-endings

**Review by rtaylorzfin — APPROVED** [review]
Looks good. I created a couple of PRs to this PR that fix the line endings issue.

File: source/org/zfin/framework/dao/VocabularyDAO.java (hunk 1 above)
Comment (cmpich) [reply source/org/zfin/framework/dao/VocabularyDAO.java:1]: Thanks.

**Pull request merged**